eva-accession-import-automation/create_fasta_from_assembly_report.sh

```diff
@@ -33,6 +33,9 @@ exit_code=0
 touch ${output_folder}/${species}_custom.fa
 touch ${output_folder}/written_contigs.txt
 
+# remove the FASTA index because it might get outdated (EVA-1581)
+rm -f ${output_folder}/${species}_custom.fa.fai
```

Comment:
will we delete the FASTA index everytime this script is executed or should we only do it when new contigs are written to the FASTA file?

Reply:
this will require a flag to check if at least one contig has been added by the script. To be on the simpler side, I would suggest to remove FAI unconditionally (just in case there were modifications to the FASTA outside of the script).

Reply:
the pipelines will generate the index anyway, and it's just some seconds. I prefer unconditional removal too

Reply:
ok, let's leave it as is